x-pack/plugins/security_solution/public/ai_insights/insight_markdown_formatter/field_markdown_renderer/get_user_flyout_panel_props.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FlyoutPanelProps } from '@kbn/expandable-flyout';
+import { TableId } from '@kbn/securitysolution-data-table';
+
+const UserPanelKey: UserPanelExpandableFlyoutProps['key'] = 'user-panel';
+
+interface UserPanelProps extends Record<string, unknown> {
+  contextID: string;
+  scopeId: string;
+  userName: string;
+  isDraggable?: boolean;
+}
+
+interface UserPanelExpandableFlyoutProps extends FlyoutPanelProps {
+  key: 'user-panel';
+  params: UserPanelProps;
+}
+
+export const isUserName = (fieldName: string) =>
+  fieldName === 'user.name' || fieldName === 'host.hostname';
```

Comment:
```suggestion
  fieldName === 'user.name';
```
Maybe leftover from copy/paste? Not sure there is a secondary user name field we need to check?